### ISSUE TITLE
no emails for new project requests

### ISSUE THREAD

**Issue by wittigue** [seek4science/seek#1420]
Today there were two new project requests in FAIRDOMHub but no emails sent to admins to approve it.

**Comment by OlgaKrebs** [comment]
yes ,  e.g. for project https://fairdomhub.org/projects/352  created today no request came

**Comment by stuzart** [comment]
yes, there is a problem with sending emails that we are trying to figure out.

**Comment by stuzart** [comment]
it's been fixed now, it was due to a library changing its default behaviour after a recent update
